ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/fileinput/v2/fileinput/_cq_dialog/.content.xml

```diff
@@ -79,8 +79,7 @@
                     </basic>
                     <validation
                         jcr:primaryType="nt:unstructured"
-                        sling:resourceType="granite/ui/components/coral/foundation/include"
-                        path="core/fd/components/form/fileinput/v1/fileinput/cq:dialog/content/items/tabs/items/validation"/>
+                        sling:resourceType="granite/ui/components/coral/foundation/container"/>
```

Comment:
You need to re-use the dialog from super resource type, wasn't the earlier change correct ? Can you also write a test case for this in the change.

Reply:
Earlier change was also correct but somehow Validation tab text was not coming hence took the reference of parent resource only. Added the test case of the Validation as well.